deepchem/feat/molecule_featurizers/mol2vec_fingerprint.py

```diff
@@ -11,187 +11,187 @@
 
 
 def _mol2alt_sentence(mol, radius):
-  """Same as mol2sentence() except it only returns the alternating sentence
-    Calculates ECFP (Morgan fingerprint) and returns identifiers of substructures as 'sentence' (string).
-    Returns a tuple with 1) a list with sentence for each radius and 2) a sentence with identifiers from all radii
-    combined.
-    NOTE: Words are ALWAYS reordered according to atom order in the input mol object.
-    NOTE: Due to the way how Morgan FPs are generated, number of identifiers at each radius is smaller
-
-    Parameters
-    ----------
-    mol : rdkit.Chem.rdchem.Mol
-    radius : float
-      Fingerprint radius
-
-    Returns
-    -------
-    list
-      alternating sentence
-    combined
-    """
-  # Copied from https://github.com/samoturk/mol2vec/blob/850d944d5f48a58e26ed0264332b5741f72555aa/mol2vec/features.py#L129-L168
-  radii = list(range(int(radius) + 1))
-  info = {}
-  _ = AllChem.GetMorganFingerprint(
-      mol, radius,
-      bitInfo=info)  # info: dictionary identifier, atom_idx, radius
+    """Same as mol2sentence() except it only returns the alternating sentence
+      Calc6ulates ECFP (Morgan fingerprint) and returns identifiers of substructures as 'sentence' (string).
+      Returns a tuple with 1) a list with sentence for each radius and 2) a sentence with identifiers from all radii
+      combined.
+      NOTE: Words are ALWAYS reordered according to atom order in the input mol object.
+      NOTE: Due to the way how Morgan FPs are generated, number of identifiers at each radius is smaller
+
+      Parameters
+      ----------
+      mol : rdkit.Chem.rdchem.Mol
+      radius : float
```

Comment:
The docstring should be formatted to follow numpy docstring style [conventions](https://numpydoc.readthedocs.io/en/latest/format.html). You can use the script [here](https://gist.github.com/gusty1g/3ed5abd147d44bca65383c4d992bb7f5) to format docstring.

Reply:
That script [https://gist.github.com/gusty1g/3ed5abd147d44bca65383c4d992bb7f5](here) is not working properly in indenting the docstring. What should I do in this case?

Reply:
That's an issue we are also fixing - that was a quick code which I wrote some time back. We can try fixing the script or fixing the docstring manually.